vignettes/custom_appearance.Rmd

```diff
@@ -90,3 +94,167 @@ lyt3 <- basic_table() %>%
 tbl3 <- build_table(lyt3, ex_adsl)
 tbl3
 ```
+
+### Top-left Materials
+
+The sequence of sequence of strings printed in the area between the column header display and the first row label (\emph{top left material}) can be modified using the `top_left()` function.
+
+```{r}
+lyt <- basic_table() %>%
```

Comment:
The primary ways of getting top-left materials are

a) in the basic_table constructor

```
lyt <- basic_table(top_left = c("hi I'm", "topleft materials"))
```

Or even more commonly, b) setting row-split label positions to "topleft" which gives the standard structure summary topleft materials, see the output of:

```
lyt <- basic_table() %>%
  split_cols_by("ARM") %>%
  split_rows_by("STRATA1", label_pos = "topleft") %>%
  split_rows_by("SEX", label_pos = "topleft") %>%
  analyze("AGE")

built_table(lyt, DM)
```

Please modify this section to include those use cases _in addition to_ the post-processing modification as you currently have